tests/helpers.bash

```diff
@@ -10,6 +10,15 @@ BUILD_NUM="${BUILD_NUMBER:-0}"
 JOB_BASE="${JOB_BASE_NAME:-local}"
 BUILD_ID="${JOB_BASE}-${BUILD_NUM}"
 
+# Prefer local build if binary file detected.
```

Comment:
but aren't the runtime tests executed against the cilium-agent started by systemd? the systemd service files specify the full path of the directory

```
Unit]
Description=cilium-docker plugin
Documentation=https://github.com/cilium/cilium
Requires=docker.service

[Service]
Environment=INITSYSTEM=SYSTEMD
Type=simple
ExecStartPre=-/usr/bin/killall -9 cilium-docker
ExecStart=/usr/bin/cilium-docker
```